core/src/main/web/app/styles/dashboard.scss

```diff
@@ -22,6 +22,11 @@
     margin-top: 40px;
     margin-bottom: 60px;
   }
+  padding-top: 76px;
```

Comment:
In the master branch the dashboard is pushed down by adding a padding-top to <body>, so that the title and menu bars do not overlap with the dashboard. However, doing this in body is awkward, and removing this padding on the electron window would involve adding a class to body (which I also think is not possible with ng-class in Angular). So I just moved this padding to dashboards.